app/controllers/sitcoms_controller.rb

```diff
@@ -4,9 +4,21 @@ class SitcomsController < ApplicationController
 
   def index
     unless params[:search] == nil
```

Comment:
This controller action is looking super fat with a ton of instance variables. Could you refactor to clean that up? If you need ideas I'm happy to provide some.